dotnet-authserver/src/TeacherIdentity.AuthServer/UserClaimHelper.cs

```diff
@@ -56,17 +58,26 @@ public async Task<IReadOnlyCollection<Claim>> GetPublicClaims(Guid userId, Func<
             return Array.Empty<Claim>();
         }
 
+        var fullName = user.MiddleName is null
```

Comment:
Not sure if you also wanted the middle name to be included in the Name claim

Reply:
Included seems best to me